README.md

```diff
@@ -19,6 +19,7 @@ give you a working calendar component.
 
 * `activeMonth` — any day within the month that you want initially displayed
 * `blockClassName` — base class name that will be used as a class prefix (see [](#class-names))
+* `descriptionText` — Text will be rendered below the month header 
```

Comment:
This looks like very specific case. Some could want to render description below, some above. I would think of more generic solution, for example:

- Allow to pass a custom header component which will be used instead of the built-one.
- Give this prop more generic name, e.g. `afterHeader`.

I would prefer first approach because it's most scalable. If you have better ideas, please bring them up.